### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by SaMnCo**
Hi guys! 

Good job on the release! I was able to deploy on AWS and now I am testing the features. I made some lightweight updates in the README file for consistency, feel free to include if you like them. 

++
Sam

### CODE REVIEW

**Comment by chuckbutler** [comment]
@SaMnCo this PR appears to be out of date. would you mind rebasing with master and fixing this up? I'd like to include your fixes in this branch

**Comment by SaMnCo** [comment]
@chuckbutler  OK I think we are good now. Sorry about that.

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 ## Overview
 
-This is a Kubernetes cluster that includes logging, monitoring, and operational
+This is a Kubernetes 1.4.0 cluster that includes logging, monitoring, and operational
```
Comment (chuckbutler) [inline README.md]: We have a badge that we're updating with the kubernetes version number. I dont know that we wish to include this inline as its another source to update.

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
 knowledge. It is comprised of the following components and features:
 
 - Kubernetes (automated deployment, operations, and scaling)
-  - Three node Kubernetes cluster with one master and two worker nodes.
+  - Four node Kubernetes cluster with one master and three worker nodes.
```
Comment (chuckbutler) [inline README.md]: Good catch

File: README.md [hunk 3]
```diff
@@ -197,8 +197,16 @@ cluster. If one is not available, you may deploy this with:
 juju config kubernetes-worker ingress=true
 ```
 
-Ingress resources are DNS mappings to your containers, routed through
-[endpoints](http://kubernetes.io/docs/user-guide/services/)
+In any case, in order to use the ingress, you will need to expose the worker service via
+
+```
+juju expose kubernetes-worker
```
Comment (chuckbutler) [inline README.md]: This was added with the last update. ~ line number  187. It looks weird having this in 2 places under the same subheading.

File: README.md [hunk 4]
```diff
@@ -256,7 +264,8 @@ nginx-ingress-controller-0f7r6   1/1       Running   0          1h
 #### List the services and endpoints
 
 ```
-$ kubectl get svc,ep
+
```
Comment (mbruzek) [inline README.md]: We can not have empty lines between a tick fence and the text or it will not render correctly on jujucharms.com

Comment (chuckbutler) [inline README.md]: +1 on catching this @mbruzek

File: README.md (hunk 1 above)
Comment (mbruzek) [inline README.md]: We should not put the version in here for the burden of having to update that for all the releases we are going to make

**Comment by chuckbutler** [comment]
As this PR has grown stale > 8 days, and no longer applies cleanly against master: i'm going to close this PR for now. Please rebase and re-submit if this change is important to you

**Pull request closed without merging**